pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -139,7 +139,7 @@ func (r *NetworkInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			LoadBalancerIPAddresses: cidr,
 			PrivateIPv4CIDRs:        nc.PrivateIPv4CIDRs,
 		}
-		updateSuccess(r, &ctx, obj, r.Client, state, nc.Name, path)
+		updateSuccess(r, &ctx, obj, r.Client, state, nc.Name, path, r.Service.GetNSXLBSPath(nc.Org, nc.NsxtProject, *createdVpc.Id, obj.Namespace))
```

Comment:
Is it possible GetNSXLBSPath() return error, or notFound?

Reply:
It's possible but I think no need to do this, since LBS and VPC is created in a batch request.
GetNSXLBSPath returns empty string means no LBS created for this VPC.